js/components/pages/PassagePage.react.js

```diff
@@ -51,11 +60,17 @@ export class PassagePage extends Component {
     return (
       <div>
         <div className="mode-controls">
-          <button className="scripture-mode" onClick={() => { dispatch(asyncChangeMode(VERSE_MODE)) }}>Single verse</button>
+          <button className="scripture-mode" onClick={() => { dispatch(asyncChangeMode(VERSE_MODE)) }}>
+            {verses[active[VERSE_MODE]]._verses[0].book.substr(0,3)+' '+formatVerseIndices(verses[active[VERSE_MODE]]._verses[0])}
```

Comment:
Trying to think about the best way this could be tested.  My first thought was to add a `shortMetadata` method to the `Passage` class, which you might call like this:

```jsx
<button className="scripture-mode" onClick={() => { dispatch(asyncChangeMode(VERSE_MODE)) }}>
  { verses[active[VERSE_MODE]].shortMetadata() }
</button>
```

Another option could be to have some kind of object that you pass the `collection`, `active` state, and `mode` to and it can create your string.  

One thing to consider is the reason I prefixed `_verses` with an `_` was to indicate it was a private method attribute. Ideally we wouldn't want to call that attribute from anywhere except the Passage class, but could add a public method access that data if needed.